charts/cnpg/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for cnpg.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Number of postgres instances to create for this cluster.
+instances: 2
+
+# Postgres image to use for the cluster.
+image:
+  repository: ghcr.io/cloudnative-pg/postgresql
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+nameOverride: ""
+fullnameOverride: ""
+
+# Configuration for the postgres cluster itself
+cluster:
+
+  # Database name and owner
+  dbname: mastodon
+  owner: mastodon
+  # Secret contains the user credentials (must be kubernetes.io/basic-auth)
+  secret: postgres-auth
+
+  # Volume size for the cluster pods
+  storage: 25Gi
+
+# Configuration for cluster backup
+backup:
+  method: plugin
+  schedule: "0 0 0 * * *"  # At midnight every day
```

Comment:
I would rather do it around 4:30am

Reply:
That makes sense, but I still think midnight is a good as a "default" chart value. We can just set it to 04:30 ourselves.